target_chains/ethereum/contracts/contracts/pyth/Pyth.sol

```diff
@@ -330,7 +330,8 @@ abstract contract Pyth is
     function getTotalFee(
```

Comment:
there's another function that you need to update 
https://github.com/pyth-network/pyth-crosschain/blob/fe75b45e918d9915a1cca4458347c64481fdda9e/target_chains/ethereum/contracts/contracts/pyth/Pyth.sol#L83